src/api/app/controllers/webui/users/rss_tokens_controller.rb

```diff
@@ -1,20 +1,30 @@
 module Webui
   module Users
     class RssTokensController < WebuiController
-      before_action :require_login
+      # TODO: Remove this when we'll refactor kerberos_auth
+      before_action :kerberos_auth
+
+      after_action :verify_authorized
 
       def create
-        token = User.session!.rss_token
-        if token
+        token = User.session ? find_or_build_rss_token : Token::Rss.new
```

Comment:
The following line achieves the same, but without an extra method: 

```suggestion
        token = authorize(Token::Rss.find_or_initialize_by(user: User.session))
```

Reply:
Neat improvement! Recently working with Rails 3.2 led me into forgetting some features 😅

Reply:
Hehehe, I know the feeling.... I once had to maintain a Rails `2.3` application using Ruby `1.8.7` in 2015-2016.... I'm having nightmares again